client/src/pages/Bathroom.js

```diff
@@ -1,31 +1,234 @@
+        <li>
+          Shop with reusable containers and bags: Glass jars for bulk items like
+          flour, seeds, nuts, spices, tea, honey, coconut oil and so on Metal
+          containers, such as Lunchbots or tiffins for meat and fish (Life
+          Without Plastic sells all sorts of them) Cloth shopping bags and
+          homemade cloth produce bags for produce and bulk foods like pasta or
+          cat food
+        </li>
         <br />
-        <li>lorem</li>
-        <br />
-        <li>Recipe</li>
-
-        <NavLink className="link" to="/homemade">
-          <button>Homemade</button>{" "}
-        </NavLink>
+        <li>
+          Zero waste means zero trash and zero recycling (because you buy so
+          little in packaging). I exempt compost from the category of waste.
+          That doesn’t mean I buy more food than I can eat or throw edible food
+          on the compost heap. In fact, even my vegetable scraps don’t make it
+          to the heap until after I’ve made broth out of them. Occasionally I
+          find the odd furry lemon or a puddle of what-had-been-parsley in the
+          back of the fridge. The good news is that compost can save the world!
+          It sucks carbon out of the air and not only that, a half-inch layer of
+          this black gold can still increase yields six years after its
+          application
+        </li>
         <br />
       </ol>
-      <NavLink to="/Reusable">&lt; Back</NavLink>
+      <NavLink className="link" to="/homemade">
+        <Button>Make your own soap in Homemade</Button>{" "}
+      </NavLink>
+      <br />
+    </div>
+  );
+
+  return (
+    <div className="kitchen-wrapper">
+      <Card class="card-container">
+        <div class="card u-clearfix">
+          <div class="card-body">
+            <span class="card-number card-circle subtle">01</span>
+            <span class="card-author subtle">John Smith</span>
+            <h2 class="card-title">Bathroom Hack #1</h2>
+            <span class="card-description subtle">
+              {" "}
+              How to Move Towards a Zero-Waste Kitchen For starters, our goal
+              here is progress, not perfection. Going completely zero-waste in
+              the kitchen is a daunting thought, but with mindfulness it is not
+              such a stretch from what we’re already doing.
+            </span>
+            <div class="card-read"></div>
+            <Button onClick={onClick}>Read more</Button>
+            {showText ? <Text /> : null}
+            <span class="card-tag card-circle subtle">C</span>
+          </div>
+          <img
+            src="https://s15.postimg.cc/temvv7u4r/recipe.jpg"
+            alt=""
+            class="card-media"
+          />
+        </div>
+        <div class="card-shadow"></div>
+      </Card>
+
+      <P className="return_home">
+        <NavLink to="/Reusable">&lt; Return to Reusable</NavLink>
+      </P>
     </div>
   );
 }
+
+const Card = styled.div`
```

Comment:
Your are mixing styled components and classNames. This sort of defeats the purpose/advantage of styled components